src/astro/databases/snowflake.py

```diff
@@ -61,9 +61,12 @@ def load_pandas_dataframe_to_table(
         if target_table.metadata:
             schema = getattr(target_table.metadata, "schema", None)
 
+        # within prep_table() we use pandas drop() function which is used when we pass 'if_exists=replace'.
+        # There is an issue where has_table() works with uppercase table names but the function meta.reflect() don't.
+        # To prevent the issue we are passing table name in lowercase.
```

Comment:
Is there a GitHub issue on the SqlAlchemy repo that you know of for this issue? If so, we can add it in the comments, and replace it once that issue is fixed

Reply:
(Not a PR blocking comment)